packages/testsuite/cypress/e2e/transaction/transactions-configuration-subsystem-transaction-jdbc.cy.ts

```diff
@@ -20,20 +20,20 @@ describe("TESTS: Configuration => Sybsystem => Transaction => Path", () => {
 
   const datasourceBase = {
     connectionUrl:
-      "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;MODE=${wildfly.h2.compatibility.mode:REGULAR}",
+      "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;MODE=\\${wildfly.h2.compatibility.mode:REGULAR}",
     driverName: "h2",
   };
 
   const nonEmptyDS = {
     ...datasourceBase,
     name: "nonEmptyDS",
-    jndiName: "java:jboss/datasources/nonEmptyDS,",
```

Comment:
It seems the last comma can be also removed.

Reply:
and the same in the previous and next definition.

Reply:
yes, it is not required. But we are using prettier format tool which is adding these commas at start of testing. commad: `npx prettier --write .`

Reply:
What do you mean? I don't understand the purpose.

Reply:
explained